k8s/elasticsearch/README.md

```diff
@@ -148,28 +148,11 @@ Configure the container images:
 
 ```shell
 TAG=6.3
-export IMAGE_ELASTICSEARCH="marketplace.gcr.io/google/elasticsearch:${TAG}"
+export IMAGE_ELASTICSEARCH="marketplace.gcr.io/google/elasticsearch"
 export IMAGE_INIT="marketplace.gcr.io/google/elasticsearch/ubuntu16_04:${TAG}"
 export IMAGE_METRICS_EXPORTER="marketplace.gcr.io/google/elasticsearch/prometheus-to-sd:${TAG}"
 ```
 
-The images above are referenced by
```

Comment:
We cannot remove this section completely. It should be fixed to provide image and tag variables separately.
Tracked in issue #847